_posts/2018-04-11-Sims3CC.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Sims3 Custom Content Creation
+date: 2018-04-11  
+categories:
+  - Projects
+description: Learning the Basics of Sims3 Modding
+image: https://github.com/LaurenCBullock/Sims3Tut/blob/master/14.PNG?raw=true
+image-sm: https://github.com/LaurenCBullock/Sims3Tut/blob/master/14.PNG?raw=true
+author: Lauren Bullock
+author-image: https://avatars3.githubusercontent.com/u/21965596?s=460&v=4
+author-bio: Second Year Game Design and Development Student at Rochester Institute of Technology
+author-email: laurencbullock@gmail.com
+author-social:
+  github: https://github.com/LaurenCBullock
+  linkedin: https://www.linkedin.com/in/lauren-bullock-1b7b49126/
+---
+
+## Summary
+
+Sims custom content has been around for a while and there is a plethora of it on the internet. But despite the quantity of content on the internet there is relatively little documentation of how to create such thing for yourself. So as a person with a background with 3D modeling this seemed like an easy pursuit. My goal was to simply create several clothing pieces to be fully function in sims3.
+##Required software
```

Comment:
Can you add newlines between paragraph text and headers, also a space between the `##` and header text